DeepSpeech.py

```diff
@@ -1234,7 +1254,7 @@ def get_next_index(self, set_name):
             else:
                 # We are a remote worker and have to hand over to the chief worker by HTTP
                 log_traffic('Asking for next index...')
-                value = int(self._talk_to_chief(PREFIX_NEXT_INDEX + set_name))
+                value = int(self._talk_to_chief(PREFIX_NEXT_INDEX + set_name, default='0'))
```

Comment:
One thing I don't understand is why the default is now `'0'` where it was before `None`